### DISCUSSION TITLE
Request: README/docs on only including this in dev

### DISCUSSION THREAD

**Discussion by Kilian** [Code-Forge-Net/Remix-Dev-Tools#7] [Ideas]
Could the README be expanded with a section explaining how to only load in the component/styles when running the dev server? I saw you tweet that that's what you were doing.

**Comment by AlemTuzlak** [comment]
Will add it as soon as I precisely figure it out too, there are a lot of remix setups and dynamic imports don't work everywhere so it's a bit trickier than just doing import("remix-development-tools", module => module)

**Comment by AlemTuzlak** [comment]
@Kilian coming tomorrow

**Comment by AlemTuzlak** [comment]
This has been implemented as of v1.0.9!
